telematic_system/scripts/log_analysis/parse_v2xhub_telematic_plugin_logs.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import json
+import pandas as pd
+import sys
+import json.decoder as decoder
+from enum import Enum
+
+
+class Column(Enum):
+    UNIT_ID= "Unit ID"
+    TOPIC= "Topic"
+    PAYLOAD_TS= "Payload Timestamp"
+    PAYLOAD_SRC= "Payload Source"
+    EVENT_NAME= "Event Name"
+    UNIT_TYPE= "Unit Type"
+
+class LogKey(Enum):
```

Comment:
Can we add a comment for why these two similar enums are needed? 
Probably a note on the dependence on other logs/scripts would work

Reply:
`Probably a note on the dependence on other logs/scripts would work` Could you clarify this?